recipes/easymt/meta.yaml

```
@@ -0,0 +1,36 @@
+{% set version = "1.0.1" %}
+package:
+  name: {{name}}
+  version: {{version}}
+
+source:
+  url: https://github.com/tang-shuyuan/EeayMT/releases/download/v{{version}}/{{name}}_v{{version}}.tar.gz
+  sha256: 5c794c74159ff2f7f45c61af307361d74eaf4ab1b40dab46095927fbf5a7867c
+
+build: 
+  number: 0
+  entry_points: 
```

yamllint

[error] trailing spaces

(trailing-spaces)